magento/pwa-studio
# New userguide index and link

## Description

This pull request updates the documentation search index and site switcher link for user guide.

## Related Issue







## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to docs page
2. Click on the waffle menu to reveal the site switcher.
3. See that Commerce User Guide now points to http://docs.magento.com/user-guide/

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
[ ] I have added tests to cover my changes, if necessary.
